launch.go

```diff
@@ -24,11 +25,18 @@ type scmPath struct {
 var mkdirAll = os.MkdirAll
 var stat = os.Stat
 var gitClone = git.Clone
+var gitSetUserName = git.SetUserName
+var gitSetUserEmail = git.SetUserEmail
 
 func (s scmPath) String() string {
 	return fmt.Sprintf("%s:%s/%s#%s", s.Host, s.Org, s.Repo, s.Branch)
 }
 
+func (s scmPath) httpsString() string {
+	host := strings.Split(s.Host, "@")[1]
```

Comment:
If the url is github.com/screwdriver-cd/launcher, this will panic. At a minimum you should add @ to the regex below so that an error will get thrown if @ doesn't exist.